src/inference/src/ie_core.cpp

```diff
@@ -497,6 +497,11 @@ class CoreImpl : public ie::ICore, public std::enable_shared_from_this<ie::ICore
         return newAPI;
     }
 
+    ie::RemoteContext::Ptr GetDefaultContext(const std::string& deviceName) override {
+        auto parsed = ov::parseDeviceNameIntoConfig(deviceName, {});
+        return GetCPPPluginByName(parsed._deviceName).get_default_context(parsed._config)._ptr;
+    }
```

Comment:
can we re-use this method / implementation in `Core::GetDefaultContext` ?

Reply:
Hi ilya @ilya-lavrenov , we need to call the get context API in plugin, icore is the interface I can use.

Reply:
I understand, but from Core we can also use ICore, but now we have 2 different implementations. Let's reuse ICore from Core

Reply:
change the Core implementation, to reuse icore function.